docs/src/main/sphinx/object-storage/file-system-cache.md

```diff
@@ -112,16 +112,14 @@ enable and configure caching for the specific catalogs.
     and set the values for `fs.cache.max-sizes` or
     `fs.cache.max-disk-usage-percentages` accordingly.
 * - `fs.cache.max-sizes`
-  - Optional, comma-separated list of maximum [data sizes](prop-type-data-size)
-    for each caching directory. Order of values must be identical to the
-    directories list. Can not be used together with
-    `fs.cache.max-disk-usage-percentages`.
+  - (Optional), comma-separated list specifying the maximum [data sizes](prop-type-data-size)
+    for each caching directory.The order of values must correspond to the order of the directories list. 
+    This option cannot be used in conjunction with `fs.cache.max-disk-usage-percentages`.
 * - `fs.cache.max-disk-usage-percentages`
-  - Optional, comma-separated list of maximum percentage values of the used disk
-    for each directory. Each value is an integer between 1 and 100. Order of
-    values must be identical to the directories list. If multiple directories
-    use the same disk, ensure that total percentages per drive remains below 100
-    percent. Can not be used together with `fs.cache.max-sizes`.
+  - (Optional), comma-separated list indicating the maximum percentage values of disk usage for each directory. 
+    Each value must be an integer between 1 and 100, and the order must match the directories list.
+    If multiple directories share the same disk, ensure the total percentages for that drive do not exceed 100%.
+    This option cannot be used in conjunction with `fs.cache.max-sizes`.
```

Comment:
```suggestion
  - Comma-separated list indicating the maximum percentage values of disk usage for each directory. 
    Each value must be an integer between 1 and 100, and the order must match the directories list.
    If multiple directories share the same disk, ensure the total percentages for that drive do not exceed 100%.
    This option cannot be used with `fs.cache.max-sizes`, though one must be set.
```

Reply:
I think I'd prefer this verbiage, it's a little more wordy, but it's way more clear (and apply this to the above property as well). We need to make it obvious that you have to set one of these two properties.

Reply:
Thats fine as well with me .. not sure we should drop the initial "Optional"